instances.tf

```diff
@@ -34,28 +34,6 @@ resource "aws_instance" "bastion_win" {
   associate_public_ip_address = false
   monitoring                  = true
 
-  user_data = <<EOF
```

Comment:
Before removing the userdata inline text worth adding the userdata.ps1 rendered file and set it to userdata